docs/docsite/rst/conf.py

```diff
@@ -231,7 +231,8 @@
     'satellite_tracking': True,
     'show_extranav': True,
     'tag_manager_id': 'GTM-PSB293',
-    'vcs_pageview_mode': 'edit'
+    'vcs_pageview_mode': 'edit',
+    'display_version': False,
```

Comment:
According to https://sphinx-rtd-theme.readthedocs.io/en/stable/configuring.html#confval-display_version this no longer exists, you should use `version_selector` instead:
```suggestion
    'display_version': False,
    'version_selector': False,
```
Edit: maybe both are needed?

Reply:
Thanks @felixfontein I think maybe both are needed. I've committed your suggestion and will take a look. I'll build this branch for the package docs on the test site.